### ISSUE TITLE
Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2822]
``` 
Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #block-block-1 > div > div:nth-child(3) > a:nth-child(1)
   └── <a class="facebook-logo" href="https://www.facebook.com/PULibrary" title="Friend us on Facebook"> <img src="/ass...</a>

 • Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #block-block-1 > div > div:nth-child(3) > a:nth-child(2)
   └── <a class="twitter-logo" href="https://twitter.com/PULibrary" title="Follow us on Twitter"> <img src="/ass...</a>

 • Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #block-block-1 > div > div:nth-child(3) > a:nth-child(3)
   └── <a class="govdocs-logo" href="http://libguides.princeton.edu/govdocs" title="Government Documents Depository"> <img src="/ass...</a>

 • Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #block-block-1 > div > div:nth-child(3) > a:nth-child(4)
   └── <a class="friends-logo" href="http://library.princeton.edu/about/friends" title="Friends of the Library"> <img src="/ass...</a>

 • Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #block-block-1 > div > div:nth-child(4) > a
   └── <a class="pu-logo" href="https://www.princeton.edu" title="Princeton University"> <img width="20...</a>
```

**Comment by christinach** [comment]
I get this error when I test with` pa11y --runner axe --runner htmlcs https://catalog.princeton.edu`
If I remove the htmlcs then it doesn't log the error.
